### ISSUE TITLE
Fails to validate ``ssl.PROTOCOL_TLSv1``

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#399]
Originally reported by: **Benoît Allard (BitBucket: [benallard](http://bitbucket.org/benallard), GitHub: @benallard?)**

----------------------------------------

The following script:

```
import ssl
ssl.PROTOCOL_TLSv1
```

Give the following error:

```
E: 2, 0: Module 'ssl' has no 'PROTOCOL_TLSv1' member (no-member)
```

This is with pylint 1.4.0, python 2.7.8.

----------------------------------------
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/399

**Comment by The-Compiler** [comment]
FWIW, I just saw the same with an up-to-date pylint/astroid in a virtualenv.

**Comment by PCManticore** [comment]
@The-Compiler thanks, I'm gonna try to reproduce it then.

**Comment by The-Compiler** [comment]
Hmm, I can't reproduce this on Debian Jessie and Ubuntu Trusty FWIW, I only see it on my own Archlinux machine.

**Comment by The-Compiler** [comment]
I just noticed on the same machine I get `no-member` errors for `http.client.OK` and other [status code constants](https://docs.python.org/3.4/library/http.client.html#http.client.HTTPS_PORT). I don't get them anywhere else (like on CI) though.

**Comment by PCManticore** [comment]
Do you have a way in which I could reproduce this myself? A docker container or something similar would be nice.

**Comment by The-Compiler** [comment]
@benallard do you still have that `Dockerfile`? Last time I tried docker I only had trouble with it, so I'd rather not mess with it again :wink:

**Comment by benallard** [comment]
From the previous comments, it looks like it was pretty straight-forward: use Debian:Jessie as base, install pylint and you're done.

**Comment by The-Compiler** [comment]
I can indeed reproduce this with this `Dockerfile`:

```dockerfile
FROM debian:jessie
RUN apt-get -y update && apt-get -y install python-pip
RUN pip install pylint
RUN echo "import ssl; ssl.PROTOCOL_TLSv1" >> test.py
CMD pylint test.py -rn
```

```
$ docker build -t docker-pylint-debian .
$ docker run docker-pylint-debian
```

By the way, I'm seeing this with Python 3.5 on my Archlinux machine, so 2.7 vs. 3.5 doesn't seem to make a difference.

**Comment by PCManticore** [comment]
Fixed in astroid (https://github.com/PyCQA/astroid/commit/10122275a6b6aa6b39d8085d07a9c1ea193d4946 for 1.4.0 branch). Will be part of a pylint 1.5.6 soonish.
